overwatch/processing/trending/manager.py

```diff
@@ -56,11 +72,27 @@ def _prepareDataBase(objName, dbPosition):  # type: (str, Persistent)-> None
             dbPosition[objName] = BTree()
 
     def createTrendingObjects(self):
-        """ADD DOC"""
+        """ It loops over subsystems and calls function that creates trending objects for each subsystem.
+
+        Args:
+            None.
+        Return:
+            None.
+        """
         for subsystem in self.parameters[CON.SUBSYSTEMS]:
             self._createTrendingObjectsForSubsystem(subsystem)
 
     def _createTrendingObjectsForSubsystem(self, subsystemName):  # type: (str) -> None
```

Comment:
For "private" function there is no need to add docstring, because no one should use it from outside